faq.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: FAQ
+permalink: /faq/
+---
+
+**Affiliations**
+- _Are you a part of the Drupal Association?_
+      No. We're a collective of volunteers.
+
+- _Didn't you have space in the Drupal Association's booth at DrupalCon Baltimore?_
+Yep. They donated it to us.
```

Comment:
Is the Yep/Nope tone intentional/playful/helpful or off-putting? I'd just stick with Yes and No, but that's me.